index.md

```diff
@@ -5,7 +5,9 @@
 layout: home
 ---
 
-PELUX is a Linux based platform for your automotive project, it's based on [Yocto](http://yoctoproject.org/) and offers an improved developer experience and a quicker way to get started with your project all the way from pixel to silicon.
+PELUX is a Linux based platform for your automotive infotainment project.
+The PELUX software architecture is optimized to secure the shortest path from silicon to pixel and thereby provide optimal performance to deliver a stunning infotainment solution.
+It's based on [Yocto](http://yoctoproject.org/) and offers an improved developer experience.
```

Comment:
The new line here doesn't do anything in markdown, it will be rendered as if it wasn't there so I think you can remove it.